packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -880,6 +884,16 @@ class PopupMenuButton<T> extends StatefulWidget {
   /// the button that was used to create it.
   final Offset offset;
 
+  /// Whether this popup menu button is interactive. Useful for disabling the
```

Comment:
The convention for doc is: one brief yet somehow comprehensive summary line, the a blank line, and then more doc.